packages/cache/src/internal/requestUtils.ts

```diff
@@ -111,7 +109,8 @@ export async function retryTypedResponse<T>(
         return {
           statusCode: error.statusCode,
           result: null,
-          headers: {}
+          headers: {},
+          error: error
```

Comment:
we should address the linter issues/test failures before we merge.

Reply:
Done